content/_kss/data/identity-lockup.json

```diff
@@ -1,8 +1,8 @@
 {
   "link": "https://decanter.stanford.edu",
-  "line1": "Finance",
-  "line2": "Program In Thermal Optics",
-  "line3": "Mechanical Engineering",
-  "line4": "Engineering",
-  "line5": "School of Humanities & Sciences"
```

Comment:
This is generated from decanter. Try editing this: https://github.com/SU-SWS/decanter/blob/master/core/src/templates/components/lockup/lockup.json or override each line in each variant through the CMS tool.

Reply:
Will edit each one for now.